src/zope/server/trigger.py

```diff
@@ -215,6 +215,8 @@ def __init__(self):
                     # I've seen on two WinXP Pro SP2 boxes, under
                     # Pythons 2.3.5 and 2.4.1.
                     # (Original commit: https://github.com/zopefoundation/ZEO/commit/c4f736a78ca6713fc3dec21f8aa1fa6f144dd82f)
+                    a.close()
```

Comment:
This file looks familiar: https://github.com/zopefoundation/ZEO/blob/4.3.1/src/ZEO/zrpc/trigger.py

Reply:
That code is gone from ZEO 5 with the port to asyncio. I don't know how far they drifted apart.